source/non_matching/fe_immersed_values.cc

```diff
@@ -45,14 +45,19 @@ namespace NonMatching
     const typename Triangulation<dim>::cell_iterator &cell)
   {
     // Check that mapping and reference cell type are compatible:
-    Assert(this->get_mapping().is_compatible_with(cell->reference_cell()),
-           ExcMessage(
-             "You are trying to call FEValues::reinit() with a cell of type " +
-             cell->reference_cell().to_string() +
-             " with a Mapping that is not compatible with it."));
+    Assert(
+      this->get_mapping().is_compatible_with(cell->reference_cell()),
+      ExcMessage(
+        "You are trying to call FEImmersedSurfaceValues::reinit() with a cell of type " +
```

Comment:
Can you try to break the line; I believe this is more than 80 characters wide despite clang-format trying all tricks? Same below.
```suggestion
        "You are trying to call FEImmersedSurfaceValues::reinit() with "
        "a cell of type " +
```

Reply:
Done!